src/saltext/vmware/modules/cluster.py

```diff
@@ -83,40 +83,48 @@ def create(name, datacenter, service_instance=None):
     return {name: True}
 
 
-def get_(name, datacenter, service_instance=None):
+def get_(cluster_name, datacenter_name, service_instance=None):
```

Comment:
Why is this `get_`? `list` has a name collision with python, so we need to use the func alias, but that shouldn't be the case for get...